src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 16px;
+
+  &__product-image {
+    padding: 16px 3px 0;
+    display: flex;
+    top: 33px;
+    left: 20px;
+    width: 160px;
+    height: 134px;
```

Comment:
The width and height of the image should be controlled by the img tag, not the container. It's recommended to set these properties in the img tag itself.